cmd/kube-proxy/app/server_test.go

```diff
@@ -57,9 +58,8 @@ func TestProxyServerWithCleanupAndExit(t *testing.T) {
 
 		proxyserver, err := NewProxyServer(options)
 
-		assert.Nil(t, err, "unexpected error in NewProxyServer, addr: %s", addr)
-		assert.NotNil(t, proxyserver, "nil proxy server obj, addr: %s", addr)
-		assert.NotNil(t, proxyserver.IptInterface, "nil iptables intf, addr: %s", addr)
+		assert.NotNil(t, err, "unable to load in-cluster configuration, KUBERNETES_SERVICE_HOST and KUBERNETES_SERVICE_PORT must be defined")
```

Comment:
No, this is not useful. You're just asserting that the test failed to be able to do what it wanted

Reply:
:upside_down_face: